lib/components/molecules/Accordion/Accordion.js

```diff
@@ -0,0 +1,41 @@
+// @flow
+/**
+ *
+ * Accordion
+ *
+ */
+import React from 'react';
+import styled from 'styled-components';
+import styles from './Accordion.style';
+import type { Node } from 'react';
+
+type Props = {
+	className? : string,
+	accordionContent? : [],
+	isOpen? : boolean,
+};
+
+const Accordion = ({ className, accordionContent, isOpen }: Props): Node => 
+	<div className={className}>
+		{accordionContent && accordionContent.map((item, index) => (
+			<details className="accordion-item" key={`accordion-${index.toString()}`} open={item.isOpen}>
+				<summary className="accordion-header">
+					{item.accordionHeader}
```

Comment:
This is still not good, the components in react should make use of declarative way of definition.